.github/workflows/build-userdocs.yml

```diff
@@ -0,0 +1,14 @@
+name: Build Aeneas user docs and test them
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test Aeneas user docs
+    runs-on: [self-hosted, linux, nix]
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
Why this? By default the clone is shallow, this makes it deep. I don't think we need that

Reply:
for building, yes, i'm dumb

Reply:
Fixed.